pip_chill/pip_chill.py

```diff
@@ -3,25 +3,42 @@
 
 import pip
 
+from utils import Distribution
```

Comment:
I had to make this `from .utils import Distribution` to make `pip-chill -v` work

Reply:
Sorry... Only saw this now.